data/json/itemgroups/Weapons_Mods_Ammo/guns.json

```diff
@@ -189,6 +189,7 @@
       { "group": "nested_TDI_10", "prob": 4 },
       { "group": "nested_draco", "prob": 20 },
       { "group": "nested_ar_pistol", "prob": 25 },
+      { "group": "nested_plr16", "prob": 25 },
```

Comment:
When I went looking on gun broker I found 1081 results for AR-15 pistols chambered in 5.56. this got narrowed down to 55 results when I searched for guns with a 7-inch long barrel as is represented in game. When I looked for PLR16s in 5.56x45mm, I only got 4 results. The numbers in the item groups should probably stand to be tweaked with these rough ratios in mind.

Reply:
Check it, and i can't say it THAT bad ratio. ~Half of them in non intermediate cartige, and there is many 12+ barrel ones (not sure, what to say about 5-5.5 inches ones, they are more similar to OA-93 imo, and there is like 200+ of them). Also found 5 PLR16's, after searching by manufacturer. Looks like, there is not 1000 to 4, but ~500-600 to 5 or more in worst case.  
Also, maybe OA-93 would probably be the most appropriate example in terms of spawnrate? On GB it's 3 OA-93 and 1 OA-98, but since PLR16 still in production, it stays in same spawnpools as AR pistol?

Reply:
Spawrate changed.